prawcore/requestor.py

```diff
@@ -47,9 +47,9 @@ def close(self):
         """Call close on the underlying session."""
         return self._http.close()
 
-    def request(self, *args, **kwargs):
+    def request(self, *args, timeout=TIMEOUT, **kwargs):
```

Comment:
It looks like this argument was already supported via `**kwargs` so I don't think there is any reason to explicitly add it.